### PR TITLE
Add PHPUnit 9 support

### PR DESCRIPTION
**Pull request by Slamdunk**
See:

1. https://github.com/sebastianbergmann/phpunit/blob/9.0.0/ChangeLog-9.0.md
1. https://github.com/paratestphp/paratest/pull/74
1. https://github.com/paratestphp/paratest/pull/241

### CODE REVIEW

File: src/Runners/PHPUnit/ExecutableTest.php [hunk 1]
```diff
@@ -53,10 +53,9 @@ abstract class ExecutableTest
      */
     protected $lastCommand = '';
 
-    public function __construct(string $path, ?string $fullyQualifiedClassName = null)
```
Comment (Slamdunk) [inline src/Runners/PHPUnit/ExecutableTest.php]: > Multiple test case classes (classes that extend TestCase) that are declared in a single sourcecode file are no longer supported

Comment (andreasschroth) [reply src/Runners/PHPUnit/ExecutableTest.php]: Imo, a quite reasonable change from PHPUnit. :-)

**Comment by Slamdunk** [comment]
I've tried this PR on our business SUT and it freeze at the end of the run.
100% the issue is not related to this PR but with one of the dozen new major release of PHPUnit and deps; still it must be pointed out.
On monday I'll try to dig further the problem.

**Comment by andreasschroth** [comment]
@Slamdunk Thanks for your feedback. I'm also sure it's not related to this PR, thus I'm going to merge it once Travis shows green in a few minutes (had some network timeout before) and will tag a new version, so people can upgrade to PHPUnit 9.

**Pull request merged**